src/praisonai-agents/tests/unit/runtime/test_registry.py

```
@@ -0,0 +1,201 @@
+    
+    # Should no longer be available
+    assert not registry.is_available(runtime_id)
+
+
+def test_multiple_runtimes():
+    """Test registering and managing multiple runtimes."""
+    runtime_ids = ["runtime1", "runtime2", "runtime3"]
+    
+    # Register multiple runtimes
+    for runtime_id in runtime_ids:
+        register_runtime(runtime_id, lambda id=runtime_id: TestRuntime(id))
```

Ruff (0.15.17)

[error] Lambda argument `id` is shadowing a Python builtin

(A006)